cmd/rsh.go

```diff
@@ -108,9 +108,19 @@ func (r *iexec) podPrompt(matchingPods []v1.Pod) error {
 	}
 
 	templates := &promptui.SelectTemplates{
-		Active:   "Namespace: {{ .Namespace | blue }} | Pod: {{ .Name | cyan }}",
+		Active:   fmt.Sprintf("Namespace: {{ .Namespace | blue }} | Pod: %s {{ .Name | cyan }}", promptui.IconSelect),
 		Inactive: "Namespace: {{ .Namespace | blue }} | Pod: {{ .Name | magenta }}",
-		Selected: "Namespace: {{ .Namespace | blue }} | Pod: {{ .Name | cyan }}",
+		Selected: fmt.Sprintf("Namespace: {{ .Namespace | blue }} | Pod: %s {{ .Name | cyan }}", promptui.IconGood),
+	}
+
+	log.Debugf("naked: %v", naked)
```

Comment:
might be better to move this down one line so it only shows the debug log if naked is true

Reply:
fixed